cookbooks/aws-parallelcluster-environment/templates/cfn_hup_configuration/cfn-hook-update.conf.erb

```diff
@@ -1,5 +1,11 @@
 [parallelcluster-update]
 triggers=post.update
+<% case node['cluster']['node_type'] -%>
+<% when 'HeadNode', 'LoginNode' -%>
 path=Resources.<%= @launch_template_resource_id %>.Metadata.AWS::CloudFormation::Init
 action=PATH=/usr/local/bin:/bin:/usr/bin:/opt/aws/bin; . /etc/parallelcluster/pcluster_cookbook_environment.sh; $CFN_BOOTSTRAP_VIRTUALENV_PATH/cfn-init -v --stack <%= @stack_id %> --resource <%= @launch_template_resource_id %> --configsets update --region <%= @region %> --url <%= @cloudformation_url %> --role <%= @cfn_init_role %>
+<% when 'ComputeFleet' -%>
+path=Resources.<%= @launch_template_resource_id %>
+action=timeout 900 <%= @update_hook_script_dir %>/cfn-hup-update-action.sh
```

Comment:
Why 900? What about making such timeout configurable from a cookbook attribute, the same way we do for cmpt bootstrap timeout?

Reply:
Debug Values I added